style.css

```diff
@@ -0,0 +1,70 @@
+
+nav {
+    margin-left: 25px;
+    margin-top:25px;
+    display: flex;
+}
+nav input{
+    height: 30px;
+    width: 500px;
+    margin-left: 125px;
+    margin-top:35px;
+}
+nav button {
+    height: 30px;
+    width: 40px; 
+    margin-left:-4px;
+    margin-top: 35px;
+}
+
+#add-video-playlist {
+    margin-left: 125px;
+    margin-top:38px; 
+}
+
+#bell{
+    margin-left: 70px;
+    margin-top:38px;
+}
+
+#profile-pic img{
+    margin-left: 70px;
+    margin-top:38px;
+    height: 50px;
+    width: 50px;
+    border-radius: 50%;
+}
+
+section {
+    margin-left: 95px;
+    margin-top:55px;
+}
+#profile-record{
+    display: flex;
+}
+#video-title{
+    margin-top: 15px;
+}
+#like{
+    margin-top: 15px;
+    margin-left: 25px;
+}
+#dislike{
+    margin-top: 15px;
+    margin-left: 15px;
+}
+#share{
+    margin-top: 27px;
+    margin-left: 25px;
+}
+#subscribe{
+    margin-top: 27px;
+    margin-left: 25px;
+}
+#review-info{
+    display: flex;
+}
+aside{
+    margin-left: 200px;
+    display: flex;
```

Comment:
- [ ] You are repeating the same rules in different parts of your CSS (like `display: flex;`). For good coding practices ([check this link](https://speckyboy.com/good-bad-css-practices)), follow the DRY rule (Don't repeat yourself).
![image](https://user-images.githubusercontent.com/53256139/89356769-33203f80-d684-11ea-8a2b-7776e5a73811.png)


  👉 Here you have an explanation of how you can implement it:

  You can use multiple classes for any HTML element, so for example,

  If you use these lines in many classes:
  ```
  color: #fff;
  background-color: #000;
  ```
  and you use it for .class1, .class2, and .class3 
  you can put them inside a single class called for example "common-class" so it will look like this:
  **CSS**
  ```
  .class1 {
    font-size: 24px;
  }
  .class2 {
    font-size: 30px;
  }
  .class3 {
    font-size: 10px;
  }
  .common-class {
    color: #fff;
    background-color: #000
  }
  ```
  and in **HTML**:
  ```
  <nav class="class1 common-class">  <!-- this will have font size of 24px, color of #fff and background-color of #000 -->
  ...
  <div class="class2 common-class">  <!-- this will have font size of 30px, color of #fff and background-color of #000 -->
  ...
  <footer class="class3 common-class">   <!-- this will have font size of 10px, color of #fff and background-color of #000 -->
  ...
  ```